packages/amplify-frontend-ios/index.js

```diff
@@ -59,18 +60,43 @@ async function executeAmplifyCommand(context) {
   await commandModule.run(context);
 }
 
+const postInitQuickStart = projectPath => {
+  const awsConfigFilePath = path.join(projectPath, 'awsconfiguration.json');
+  const amplifyConfigFilePath = path.join(projectPath, 'amplifyconfiguration.json');
+  const configJsonObj = {};
+  const configJsonStr = JSON.stringify(configJsonObj);
+  if (!fs.existsSync(awsConfigFilePath)) {
+    fs.writeFileSync(awsConfigFilePath, configJsonStr);
+  }
+
+  if (!fs.existsSync(amplifyConfigFilePath)) {
+    fs.writeFileSync(amplifyConfigFilePath, configJsonStr);
+  }
```

Comment:
Can you use the `JSONUtilities.writeJson()` utility method from the `amplify-cli-core` module.

Reply:
Sure, I will do